YokDeck.lua

```diff
@@ -81,3 +81,91 @@ for _, suit in ipairs(suits) do
 		posStyle = "deck"
 	}
 end
+
+function set_up_wild_card_override()
+	local custom_enhancers_key = "yokdeck_enhancers"
+	local custom_enhancers_path = "Enhancers.png"
+
+	SMODS.Atlas {
+		key = custom_enhancers_key,
+		px = 71,
+		py = 95,
+		path = custom_enhancers_path,
+		prefix_config = {key = false}
+	}
+	
+	function using_alt_deck() 
+		for _, suit in ipairs(suits) do
+			local suit_key = suit:gsub("^%l", string.upper)
+			
+			if G.SETTINGS.CUSTOM_DECK.Collabs[suit_key] ~= "yokdeck_"..suit..skin_key_alt then
```

Comment:
It would be better to read the mod prefix set in the .json somehow but there's no obvious way to do that from what I can see.